frontend/src/metabase/nav/containers/MainNavbar/DataAppNavbarView.tsx

```diff
@@ -41,22 +48,30 @@ function DataAppNavbarView({ dataApp, items, selectedItems }: Props) {
 
   return (
     <SidebarContentRoot>
-      <SidebarSection>
-        <SidebarHeadingWrapper>
-          <SidebarHeading>{dataApp.collection.name}</SidebarHeading>
-        </SidebarHeadingWrapper>
-        <ul>
-          {appPages.map(page => (
-            <PaddedSidebarLink
-              key={page.id}
-              url={Urls.dataAppPage(dataApp, page)}
-              isSelected={dataAppPage?.id === page.id}
-            >
-              {page.name}
-            </PaddedSidebarLink>
-          ))}
-        </ul>
-      </SidebarSection>
+      <div>
+        <SidebarSection>
+          <SidebarHeadingWrapper>
+            <SidebarHeading>{dataApp.collection.name}</SidebarHeading>
+          </SidebarHeadingWrapper>
+          <ul>
+            {appPages.map(page => (
+              <PaddedSidebarLink
+                key={page.id}
+                url={Urls.dataAppPage(dataApp, page)}
+                isSelected={dataAppPage?.id === page.id}
+              >
+                {page.name}
+              </PaddedSidebarLink>
+            ))}
+          </ul>
+        </SidebarSection>
+        <div>
+          <DataAppNewButton
+            icon="add"
+            onClick={onNewPage}
+          >{t`Add new page`}</DataAppNewButton>
```

Comment:
with this button here, what does the `+` button below do?

![Screen Shot 2022-08-26 at 1 12 04 PM](https://user-images.githubusercontent.com/30528226/186975395-71a78ae2-392a-4b62-adb5-29e133af4c9d.png)

Reply:
It's supposed to be an entry point to create a new action. However, we're thinking about removing the "Add new page" button and only keep this "+" at the bottom. We'll also need some UI to add arbitrary links in the sidebar, so this "+" might just group everything we need. But let's keep it like that for now until we're certain